assets/js/Sign.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class Sign extends Component {
+  render() {
```

Comment:
it's common to do something like `const { signId, lineOne, lineTwo } = this.props` at the top of a render declaration, and then you could just do {signId} below...stylistic choice though